python/ray/data/tests/test_stats.py

```diff
@@ -990,14 +992,14 @@ def test_streaming_stats_full(ray_start_regular_shared, restore_data_context):
 * Output num rows: N min, N max, N mean, N total
 * Output size bytes: N min, N max, N mean, N total
 * Tasks per node: N min, N max, N mean; N nodes used
-* Extra metrics: {STANDARD_EXTRA_METRICS}
+* Extra metrics: {STANDARD_EXTRA_METRICS.replace(": Z", ": N")}
```

Comment:
should we alternatively call the `.replace()` on the entire string? instead of overriding `Z` with `N` both here and below?